### ISSUE TITLE
Launching codelite workspace from a shell does not open previously opened files in workspace

### ISSUE THREAD

**Issue by uffejakobsen** [eranif/codelite#2545]
Launching codelite workspace from a shell does not open previously opened files in workspace

When launching a specific codelite workspace from a shell
```
# codelite somename.workspace  
```
Codelite opens the workspace - but does not open the files that was previously opened in the last workspace session.
In fact it shows the workspace quick start matrix buttons on the startup screen - as if no workspace have been loaded yet (except that a workspace is loaded and the title bar shows that is has been loaded)

In other words codelite does not do the same when launching a workspace from a shell - compared to when selecting and opening the same workspace from with codelite itself.

Seen on CL-14.0.0 @ archlinux

**Comment by uffejakobsen** [comment]
This is a preventive action - this issue is still valid.
I'm making this comment to prevent the "stale bot" from closing it
It seems to be a bad idea to have en automatic bot close issues

**Comment by uffejakobsen** [comment]
This is a preventive action - this issue is still valid.
I'm making this comment to prevent the "stale bot" from closing it
It seems to be a bad idea to have en automatic bot close issues

**Comment by uffejakobsen** [comment]
un-stale comment

**Comment by AJenbo** [comment]
PHP Workspace on Ubuntu 20.10 using CodeLite 14.0.4 self compiled:
![image](https://user-images.githubusercontent.com/204594/97019127-370b8180-1550-11eb-8c93-6f8660b7c931.png)

**Comment by uffejakobsen** [comment]
Manual "anti-stale-bot" comment to drag this issue out of stale state...

Could someone with the right level of access please put a "bug" label or something on this issue ? thanks
Hmm it seems that the "bug" label is already there...

**Comment by AJenbo** [comment]
I already put the bug label on it, but it appears that stale bot doesn't see that one as important.

**Comment by uffejakobsen** [comment]
Manual "anti-stale-bot" comment to drag this issue out of stale state...

**Comment by uffejakobsen** [comment]
This issue is still valid.
This is a manual "anti-stale-bot" comment to drag this issue out of stale state... :-/

**Comment by uffejakobsen** [comment]
Manual "anti-stale-bot" comment to drag this issue out of stale state...

5 days pre-warning is a little short - as I've been on vacation
